scripts/ci-set-org-vars.sh

```
@@ -331,6 +335,84 @@ gitlabSetVar() {
   echo "$result" | jq --compact-output 'del(.description, .key, .value, .variable_type)'
 }
 
+# Functions to support Jenkins
+add_secret() {
+    local id=$1
+    local secret=$2
+
+    local json=$(cat <<EOF
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)